### ISSUE TITLE
[proposal] support of dumping of GrallVM IR graph using "standard" system properties.

### ISSUE THREAD

**Issue by andrii0lomakin** [beehive-lab/TornadoVM#523]
Currently TornadoVM does not dump IR Graph when system properties such as  `-Djdk.graal.Dump`, '-Djdk.graal.DumpOnError' and so on are provided. 

Usage of such properties will allow the use of an Ideal Graph Visualizer (IGV) for the development of different Phases and tracking of bugs.

**Comment by andrii0lomakin** [comment]
It is a very simple change to implement and quite a handy tool for me, so I will do that if there are no objections.

**Comment by jjfumero** [comment]
We have `tornado --igv YourProgram`. We use it quite frequently. You need to have IGV open in background, and it will dump all intermediate compilation phases in IGV.

**Comment by andrii0lomakin** [comment]
Yeah, found it. Thank you. Works like a charm. Did not find it there https://tornadovm.readthedocs.io/en/latest/flags.html . Let me keep this issue. I will add changes in the documentation.

**Comment by andrii0lomakin** [comment]
@jjfumero JFYI, I am using https://github.com/Shopify/seafoam , it is less feature complete but is more usable IMHO.
